src/containers/LearningResourcePage/components/LearningResourceForm.jsx

```diff
@@ -194,7 +196,9 @@ const LearningResourceForm = props => {
                 handleSubmit(formik, saveAsNewVersion);
               }}
               entityStatus={article.status}
-              entityType={'Article'}
+              getStateStatuses={fetchStatusStateMachine}
+              validateEntity={validateDraft}
+              isArticle={true}
```

Comment:
trur ikkje du treng true så lenge isArticle er spesifisert.
```suggestion
              isArticle
```